Automattic/wp-calypso
# Hide post editor sidebar on showLaunchpad

Related to https://github.com/Automattic/dotcom-forge/issues/2183

## Proposed Changes

* Hides the post editor sidebar when `showLaunchpad=true` is in the URL.


## Testing Instructions



* Apply this branch on your local Calypso
* Make sure `widgets.wp.com` is sandboxed.
* Follow the [wpcom-block-editor sync flow](https://github.com/Automattic/wp-calypso/tree/trunk/apps/wpcom-block-editor#dev-workflow). In summary: 
  - `cd apps/wpcom-block-editor/src`
  - `yarn dev --sync`
* Create a new post using the onboarding flag: https://{domain}/wp-admin/post-new.php?showLaunchpad=true
* The post editor sidebar should not show
* Make sure the post editor sidebar shows when showLaunchpad is missing.

## Pre-merge Checklist
